scripts/packages

```diff
@@ -17,6 +17,7 @@ ifplatform
 letltxmacro
 luacode
 luatex85
+mhchem
```

Comment:
Between `mfirstuc` and `minted` if you don't mind to respect the alphabetical order. :)

Reply:
Indeed ^^